src/style-compiler/load-postcss-config.js

```diff
@@ -0,0 +1,47 @@
+var load = require('postcss-load-config')
```

Comment:
Loading postcss config is not in scope of this project.

I'll remove this file.